app/controllers/admin/spaces_controller.rb

```diff
@@ -28,11 +28,13 @@ def create_sub_space
           name: params[:name],
           space: Space.find(params[:space_id])
         )
+      sub_space.approval_required =
+        params["/admin/spaces"][:approval_required] == "1" ? true : false
```

Comment:
Why `params["/admin/spaces"]`

Reply:
That's how the parameter manifests itself when coupled with the styling inside the form, line 215 in admin/spaces/edit

Reply:
Try using a `checkbox_tag` then, it'll be cleaner